jobserver/context_processors.py

```diff
@@ -73,6 +73,11 @@ def nav(request):
     is_active = functools.partial(_is_active, request)
 
     items = [
+        NavItem(
+            name="Analyses",
```

Comment:
Maybe this should be called `Interactive`, instead? Then it's more consistent with where we've used Interactive in other places. I like having this page in this location though, it seems like it will be very useful to people.

Reply:
Actually I think this is going to get confusing if we switch to Interactive!

We currently have:

* `/analyses/` -> list of requested analyses for the current user
* `/org/project/interactive/` -> interactive form
* `/org/project/analyses/` -> list of requested analyses for the project
* `/org/project/workspace/analyses/` -> list of requested analyses for the workspace (I'm adding this one based on your comment below)

I think we use Interactive for the nav item we might cause confusion with the URL not matching the name?

Reply:
Interactive in this case is a noun rather than an adjective, just to be picky! Might be worth sharing some screenshots and getting other people's opinions though.

Are the `*/analyses/` pages going to just have lists of requested analyses or are they going to be reports (whether pending, released or published) or both?

I think we've talked with Catherine about having `/org/project/interactive/` show a list of previous requests (or reports) and have a button at the top to create a new analysis request, so maybe we don't need `/org/project/analyses/` as well?

Reply:
Dictionary says otherwise!  This is the literally only way I will ever know what type a word is.

The `*/analyses/` pages were going to show lists of existing AnalysisRequests, based on the context of the viewer.  There _might_ be a report on the page the user clicks through to, but there isn't a guarantee of one.

Vaguely, but I'll pivot that page to show those for this PR.

Reply:
Change of plan, we need some more planning around URLs to display a list of Analyses on the `*/interactive` page, because it changes around how the form works.  Lets discuss at the next tech meeting for OSI!

For now I'll drop the project-specific page and add a workspace-specific one.

Reply:
> Dictionary says otherwise! This is the literally only way I will ever know what type a word is.

This is totally not the slightest bit important, but while interactive is an adjective, **Interactive** is the name of a thing, so it's a proper noun. It's also totally possible to turn [adjectives into nouns](https://en.wikipedia.org/wiki/Nominalized_adjective) in English (yes, I did need to google that).